src/mongo/db/repl/member_config_test.cpp

```diff
@@ -75,6 +76,14 @@ TEST(MemberConfig, ParseFailsWithMissingIdField) {
                                 &tagConfig));
 }
 
+TEST(MemberConfig, ParseFailWithJustHostInternal) {
```

Comment:
We should add tests that verify that if hostInternal is not set, it is filled in by the host field, def. for compatibility and mixed versions.